lang/nl.yml

```diff
@@ -1,52 +1,43 @@
-    UseElementalGrid:
-      Label: 'Gebruik elemental grid'
-      Description: 'Sla de pagina eerst op nadat je deze instelling aanpast.'
-  Element:
-    Title:
-      Title: 'Titel'
-      Tag: 'Titel tag'
-      DisplayAs: 'Weergeven als'
-      Displayed: 'Weergeven'
-  Column:
-    Label: 'Kolom'
-    Default: 'Standaard'
-    None: 'Geen'
-    Viewport:
-      XS: 'Extra klein (voorbeeld: smartphone)'
-      SM: 'Klein (voorbeeld: staande tablet)'
-      MD: 'Medium (voorbeeld: liggende tablet)'
-      LG: 'Groot (voorbeeld: normale desktop)'
-      XL: 'Extra groot (voorbeeld: full HD monitor)'
-    Size:
-      XS: 'Formaat extra klein'
-      SM: 'Formaat klein'
-      MD: 'Formaat medium'
-      LG: 'Formaat groot'
-      XL: 'Formaat extra groot'
-    Offset:
-      XS: 'Inspring extra klein'
-      SM: 'Inspring klein'
-      MD: 'Inspring medium'
-      LG: 'Inspring groot'
-      XL: 'Inspring extra groot'
-    Visibility:
-      XS: 'Zichtbaarheid extra klein'
-      SM: 'Zichtbaarheid klein'
-      MD: 'Zichtbaarheid medium'
-      LG: 'Zichtbaarheid groot'
-      XL: 'Zichtbaarheid extra groot'
-      Visible: 'Zichtbaar'
-      Hidden: 'Onzichtbaar'
-  Row:
-    Label: 'Rij'
-    CustomRowClasses: 'Aangepaste rij classes'
-    IsFluid: 'Deze rij gebruikt de volledige breedte'
-  Section:
-    Section: 'Sectie'
-    CustomSectionClasses: 'Aangepaste sectie classes'
+  TheWebmen\ElementalGrid\Extensions\ElementalPageExtension:
+    USE_ELEMENTAL_GRID: 'Gebruik elemental grid'
+    USE_ELEMENTAL_GRID_DESCRIPTION: 'Sla de pagina eerst op nadat je deze instelling aanpast.'
+  TheWebmen\ElementalGrid\Extensions\BaseElementExtension:
+    TITLE: 'Titel'
+    TITLE_TAG: 'Titel tag'
+    DISPLAY_AS: 'Weergeven als'
+    DISPLAYED: 'Weergeven'
+    COLUMN: 'Kolom'
+    DEFAULT: 'Standaard'
+    NONE: 'Geen'
+    VIEWPORT_XS: 'Extra klein (voorbeeld: smartphone)'
+    VIEWPORT_SM: 'Klein (voorbeeld: staande tablet)'
+    VIEWPORT_MD: 'Medium (voorbeeld: liggende tablet)'
+    VIEWPORT_LG: 'Groot (voorbeeld: normale desktop)'
+    VIEWPORT_XL: 'Extra groot (voorbeeld: full HD monitor)'
+    SIZE_XS: 'Formaat extra klein'
+    SIZE_SM: 'Formaat klein'
+    SIZE_MD: 'Formaat medium'
+    SIZE_LG: 'Formaat groot'
+    SIZE_XL: 'Formaat extra groot'
+    OFFSET_XS: 'Inspring extra klein'
+    OFFSET_SM: 'Inspring klein'
+    OFFSET_MD: 'Inspring medium'
+    OFFSET_LG: 'Inspring groot'
+    OFFSET_XL: 'Inspring extra groot'
+    VISIBILITY_XS: 'Zichtbaarheid extra klein'
+    VISIBILITY_SM: 'Zichtbaarheid klein'
+    VISIBILITY_MD: 'Zichtbaarheid medium'
+    VISIBILITY_LG: 'Zichtbaarheid groot'
+    VISIBILITY_XL: 'Zichtbaarheid extra groot'
+    VISIBLE: 'Zichtbaar'
+    HIDDEN: 'Onzichtbaar'
   TheWebmen\ElementalGrid\Models\ElementRow:
+    LABEL: 'Rij'
+    CUSTOMROWCLASSES: 'Aangepaste rij classes'
+    ISFLUID: 'Deze rij gebruikt de volledige breedte'
+    SECTION: 'Sectie'
+    CUSTOMSECTIONCLASSES: 'Aangepaste sectie classes'
```

Comment:
Above snake-case is used (TITLE_TAG).
For readability and consistency I suggest to format this to;

```
    CUSTOM_ROW_CLASSES: 'Aangepaste rij classes'
    IS_FLUID: 'Deze rij gebruikt de volledige breedte'
    SECTION: 'Sectie'
    CUSTOM_SECTION_CLASSES: 'Aangepaste sectie classes'
```